src/api/comments.ts

```diff
@@ -0,0 +1,20 @@
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+export const getComments = (postId: Comment['postId']) => {
+  return client.get<Comment[]>(`
+/comments?postId=${postId}`);
+};
+
+export const createComment = ({
+  postId,
+  name,
+  email,
+  body,
+}: Omit<Comment, 'id'>) => {
+  return client.post<Comment>(`/comments`, { postId, name, email, body });
+};
+
+export const deleteComment = (commId: Comment['id']) => {
```

Comment:
The parameter name `commId` is inconsistent with the rest of the codebase where `commentId` might be more appropriate for clarity and consistency.